app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+
+    ```js
+    ready: function() {
+      this.scopeSubtree(this.$.container, true);
+    }
+    ```
+
+    `containerNode` is the root node of the tree you wish to scope. Setting
+    the second argument to `false` scopes the specified node and descendants
+    **once.** Setting it to `true` enables a mutation observer that applies CSS
+    scoping whenever `containerNode` or any of its descendants are modified.
+
+**Not for use on Polymer elements.** If the subtree that you scope
+contains any Polymer elements with local DOM, `scopeSubtree` will
+cause the descendants' local DOM to be styled incorrectly.
+{.alert .alert-error}
+
+## Styling at the document level
+	
+Browsers that implement the current Shadow DOM v1 specifications will automatically encapsulate styles, scoping them to the elements in which they were defined.
+
+Some browsers have not implemented the Shadow DOM v1 specifications. To make sure your apps and elements display correctly in these browsers, you'll need to use `custom-style` to ensure that styling information doesn't "leak" into the local DOM of your elements.
+
+The `custom-style` extension enables a set of polyfills that ensure that styles in your apps and elements behave as you would expect from the Shadow DOM v1 specifications, even in browsers that don't implement these specifications.
+
+`custom-style` is an extension of the native `<style>` element. To ensure that your styles behave according to the Shadow DOM v1 specifications in all browsers, use `custom-style` when you define *document-level* styles.
+
+*Note: You should only use `custom-style` to define styles for the main document. To define styles for an element's local DOM, just use a `<style>` block.*
+
+### Examples
+
+In the first code sample, the style for the `p` element “leaks” into Paragraph B in browsers that haven’t implemented the Shadow DOM v1 specs. In the second code sample, the developer has used `custom-style` to wrap the style block, preventing this leak.
+
+[See it on Plunker](http://plnkr.co/edit/0o1zuMHgmt4novf2DS8z?p=preview)
+
+```html
+  <style>
+    p {
+      color: red;
+    }
+  </style>
+  <p>Paragraph A: I am in the main DOM. I am red.</p>
+  <dom-module id="x-foo">
+    <template>
+      <p>Paragraph B: I am in the local DOM of x-foo. 
+         If your browser implements the Shadow DOM v1 specs, 
+         I am black; otherwise, I'm red.
+      </p>
+    </template>
+  </dom-module>
+  <x-foo></x-foo>   
+```	
+
+[See it on Plunker](http://plnkr.co/edit/yiD9XWPHaMjHaGGwu4V9?p=preview)
+```html
+  <custom-style>
+  <style>
+      p {
+        color: red;
+      }
+    </style>
+  </custom-style>
+  <p>Paragraph A: I am in the main DOM. I am red.</p>
+  <dom-module id="x-foo">
+    <template>
+      <p>Paragraph B: I am in the local DOM of x-foo. I am black on all browsers.</p>
+    </template>
+  </dom-module>
+  <x-foo></x-foo>   
+```
+  
+### Syntax and compatibility
+
+#### Polymer 2.x
```

Comment:
Style wise, for consistency when captioning a code sample you can use:

`Polymer 2.x {.caption}`